src/main/java/nl/cherement/jak/service/CardService.java

```diff
@@ -22,6 +29,23 @@ public CardService(CardRepository repository) {
         return this.repository.findByColumnId(id);
     }
 
+    @Override
+    public CardEntity save(Authentication user, CardEntity o) {
+        Optional<CardEntity> cardOptional = repository.findById(o.id);
+        if (cardOptional.isPresent()) {
+            CardEntity cardEntity = cardOptional.get();
+            if (cardEntity.column.id != o.column.id) {
```

Comment:
Waarom duwen we m niet gelijk de repository in?

Reply:
zie andere comment